### PR TITLE
Add option to show contents of collapsed comments

### PR DESCRIPTION
**Pull request by twizmwazin**
Basic functionality is working, but still a few TODOs on this:

- ~No indication if a comment is collapsed since the whole body is shown. Boost uses a little +N icon in the top bar.~
- ~Comments folding isn't animated currently~
- ~Settings migrations needs to be tested~

### CODE REVIEW

**Comment by dessalines** [comment]
Looking good so far! You might need to ping me whenever you need me to review.

**Comment by twizmwazin** [comment]
Alright @dessalines, this should be ready!

**Comment by a1studmuffin** [comment]
FYI I've also made a PR that fixes the comment collapsing feature I added from closing the top-level comment, which felt a bit weird/clumsy: https://github.com/dessalines/jerboa/pull/446

I don't mind which way we go though, just thought I'd throw it out there. :)

File: app/src/main/java/com/jerboa/ui/components/common/TimeAgo.kt [hunk 1]
```diff
@@ -103,3 +120,33 @@ fun ScoreAndTimePreview() {
...
+            Box(
+                modifier = Modifier
+                    .clip(RoundedCornerShape(2.dp))
+                    .background(MaterialTheme.colorScheme.secondary)
+                    .padding(horizontal = SMALL_PADDING),
+            ) {
+                Text(
+                    text = "+$descendants",
+                    style = MaterialTheme.typography.bodyMedium,
+                    color = Color.White,
```
Comment (dessalines) [inline app/src/main/java/com/jerboa/ui/components/common/TimeAgo.kt:141]: Use an m3 theme color.

**Comment by twizmwazin** [comment]
I'm testing the color fix right now and it should be posted in minutes.

Regarding the two PRs, I obviously have a preference towards mine. However, @a1studmuffin, I wouldn't mind adding a toggle to enable the behavior in your PR so that users can choose. My design is pretty much a one-to-one copy of boost's behavior.

**Review by dessalines — APPROVED** [review]
Thanks, looks good!

Once it passes CI we can merge, run the `./gradlew formatKotlin`

**Review by dessalines — APPROVED** [review]
Thanks, looks good!

Once it passes CI we can merge, run the `./gradlew formatKotlin`

**Pull request merged**